package-system/OpenEXR/build_config.json

```diff
@@ -0,0 +1,119 @@
+{
+   "git_url":"https://github.com/AcademySoftwareFoundation/openexr.git",
+   "git_tag":"RB-2.3",
+   "package_name":"ilmbase",
+   "package_version":"2.3.0-rev4",
+   "package_url":"https://github.com/03de/OpenEXR",
+   "package_license":"Apache-2.0",
+   "package_license_file":"LICENSE",
+   "cmake_find_target":"Findilmbase.cmake",
+   "Platforms":{
+      "Windows":{
+         "Windows":{
+            "cmake_find_template":"Findilmbase.cmake.Windows",
+            "cmake_generate_args_debug":[
+               "-G",
+               "\"Visual Studio 16 2019\"",
+               "-A",
+               "x64",
+               "-T",
+               "host=x64",
+               "-DOPENEXR_BUILD_PYTHON_LIBS=OFF",
+               "-DOPENEXR_BUILD_TESTS=OFF",
+               "-DOPENEXR_BUILD_UTILS=OFF",
+               "-DOPENEXR_BUILD_SHARED=OFF",
+               "-DOPENEXR_BUILD_STATIC=ON",
+               "-DCMAKE_INSTALL_PREFIX=\"../../ilmbase-windows/ilmbase\"",
+               "-DCMAKE_BUILD_TYPE=Debug",
+               "-DZLIB_ROOT=../../../zlib-windows/zlib"
```

Comment:
This is a situation that we may need to address in the script. We may need to either use a custom script instead of the using the cmake arguments here, so that we would:

1. Sync OpenEXR (this can still be done by the script)
2. Have the custom script do a git clone for zlib based on the appropriate tags, etc
3. Build zlib for that platform first, then apply the zlib argument here
I'm not sure we want to do this long term (relying on the README), but I guess its acceptable for now

Reply:
I'd like to fix this but I don't think its in the scope of adding ILMBase to fix it.

Ideally we'd be able to declare dependencies on other packages here and it fetches and unpacks it using the normal 3p system.  It does not need to build the other package, right?  We already have auto build scripts for zlib too, which means other option is for it to build the package (using the command in the host file) and use its "output folder" (also listed in the host file) as the source of it.

Reply:
That approach sounds good, but dependencies would need to be declared in main build scripts and potentially can be declared in the 'package_build_list*.json' files.